Sources/PlaybackSDK/Player Plugin/BitMovinPlugin/BitmovinPlayerPlugin.swift

```diff
@@ -73,7 +73,7 @@ public class BitmovinPlayerPlugin: VideoPlayerPlugin, ObservableObject {
         if self.player == nil {
             let player = PlayerFactory.createPlayer(
                 playerConfig: playerConfig,
-                analytics: self.createAnalyticsConfig(analyticsViewerId: analyticsViewerId)
+                analytics: self.createAnalyticsConfig(analyticsViewerId: analyticsViewerId, videoDetails: videoDetails.first)
```

Comment:
You pass the first element of the videoDetails only, but in that way the `videoId` and the `title` will be always the first one. I think you have to change the logic in order to create/pass the right `videoDetails` for every item in the playlist. That approach will work only if the user plays a single video.